src/pages/CraftPageCollectionPage/CraftPageCreationModal.tsx

```diff
@@ -110,11 +112,12 @@ const CraftPageCreationModal: React.VFC<
   const { insertAppPage } = useMutateAppPage()
   const [loading, setLoading] = useState(false)
   const [currentTemplate, setCurrentTemplate] = useState<'A' | 'B' | 'C' | 'empty'>('empty')
-
-  const [pageInfo, setPageInfo] = useState<{ pageName: string; path: string }>(
-    defaultPageInfo || { pageName: '', path: '' },
+  const [pageId, setPageId] = useState<string>()
```

Comment:
I think this state is unnecessary. We can get the `pageId` after `insertAppPage` and then route to it.

Reply:
https://github.com/urfit-tech/lodestar-app-admin/pull/354#discussion_r1295337209